packages/testing/config/jest/web/jest.setup.js

```diff
@@ -3,26 +3,39 @@
 require('@testing-library/jest-dom')
 require('whatwg-fetch')
 
-const { getProject } = require('@redwoodjs/structure')
+const { loadCellMocks } = require('@redwoodjs/testing/dist/web/loadCellMocks')
 const {
   startMSW,
   setupRequestHandlers,
+  closeServer,
   mockGraphQLMutation,
   mockGraphQLQuery,
   mockCurrentUser,
-} = require('@redwoodjs/testing/web')
+} = require('@redwoodjs/testing/dist/web/mockRequests')
 
 global.mockGraphQLQuery = mockGraphQLQuery
 global.mockGraphQLMutation = mockGraphQLMutation
 global.mockCurrentUser = mockCurrentUser
 
-const project = getProject()
+// NOTE: for performance reasons, we're not using rwjs/internal here
+// This way we can make sure only the imports we require are loaded
+const cellMocks = loadCellMocks(global.__RWJS_TESTROOT_DIR)
```

Comment:
Maybe instead of setting a global, we could just piggyback off Jest setting the `process.cwd` to `rootDir`? (Which we set as the root of the RW project in a project's root `jest.config.js`) And just do `path.join` from there?

```js
const cellMocks = loadCellMocks(path.join(process.cwd(), 'web', 'src'))
```

Reply:
The problem with that approach is that it looks us in to a hard coded 'web' folder - if we try to keep paths consistently imported from internal, we won't break things when we decide to support multiple web sides (or different folder names). Ideally jest would actually provide the directory it's actually running in, but it doesn't look like it does! 

getPaths is already required in this file anyway for the other settings, and to me it looks like the impact here is minimal.

Reply:
We can stop setting `rootDir` to `'../'` if we want Jest to say we're running in the web folder. We're telling it to run in the root of the project for both sides:

https://github.com/redwoodjs/redwood/blob/07400f7c223ccb05ce60d7c9ad103e859857bedd/packages/create-redwood-app/template/web/jest.config.js#L4

I don't think `getPaths` is required in this file; otherwise we wouldn't be setting a global? Did you mean `jest-preset.js`?

And yeah I'm not talking about the impact to how tests run. Mostly just saying I feel like there should be an easier way to do this than setting a global.

Reply:
I don't fully remember, but I think the reason the root is set to the top level is so that jest works without our cli (so that extensions run, zero config).

Reply:
Oh there's a comment right above it:
>   // To make sure other config option which depends on rootDir always
>  // use correct path, for example, coverageDirectory

Reply:
Discussed in a call; the way jest works resetting the env per worker, this makes sense.